apps/store/src/pages/products/[product].tsx

```diff
@@ -37,7 +37,7 @@ export const getServerSideProps: GetServerSideProps<NextPageProps> = async (cont
   const { countryCode } = getCountryByLocale(locale)
 
   try {
-    const apolloClient = initializeApollo()
+    const apolloClient = initializeApollo(undefined, req, res)
```

Comment:
How do you feel about changing `initializApollo` function signature to accept an object instead of a list of arguments like the following:

```suggestion
    const apolloClient = initializeApollo({ req, res })
```

I understand that TS will help us to understand what is that first argument that we're passing `undefined` to but by using an object it would be even clear and parameter order wouldn't matter.

Reply:
yep I think that's a good idea -- will add it to the backlog!